### ISSUE TITLE
Transfer learning using a different learning rate

### ISSUE THREAD

**Issue by ZGuangxuan** [unit8co/darts#1109]
Hello,

I hope to change the learning rate when fitting a  pretrained LSTM model in a new data set. But I failed to modify it. Is that possible?Could you help me with this problem? It is a bit urgent. Thanks!

**Comment by madtoinou** [comment]
Hi,

Sorry for the delay, it appears that your issue slipped through.

The linked PR should solve your issue but in the meantime, you could adapt the code present in the ([gist](https://gist.github.com/solalatus/e6329879c5c9479ce60af9f6b0e22bb4)) shared by @solalatus to modify the learning rate of the optimizer and increase the number of epochs before calling `fit` with the updated new pytorch lightning trainer.
